### PR TITLE
waiting for code review

### PR DESCRIPTION
**Pull request by ViriCruz**

### CODE REVIEW

**Comment by AdnanAfsari** [comment]
Adnan Afsari is working on it

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    attempt = 0
+    puts "Turn #{@game.turn}, #{player.name} choose a position from 1 to 9 in the table"
+    @game.board.display_board
+
+    until available
+      puts "Position not available. #{player.name} choose another position from 1 to 9" if attempt > 0
+      position = gets.chomp
+      case @game.valid_number(position)
+      when 1
+        puts "valid position. checking if it's available"
```
Comment (AdnanAfsari) [inline bin/main.rb:55]: you can remove this, its not needed i guess :)

File: bin/main.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        position = @game.convert_to_number(position)
+        available = @game.check_availability?(position)
+        attempt += 1 unless available
+        if available
+          attempt = 0
+          puts "position is available"
+          puts "updating game"
```
Comment (AdnanAfsari) [inline bin/main.rb]: also this

File: bin/main.rb [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          puts "displaying board updated"
```
Comment (AdnanAfsari) [inline bin/main.rb]: also this

**Review by AdnanAfsari — APPROVED** [review]
Hello @ViriCruz Congratulation 👍 

Everything looks perfect and complete , I was also able to play the game in my local machine :)

You can merge it now .

Happy Coding

File: bin/main.rb (hunk 1 above)
Comment (ViriCruz) [reply bin/main.rb:55]: Hii Adnan, do you mean the line 55 > puts "valid position. checking if it's available"?
thanks for checking my code.

Comment (AdnanAfsari) [reply bin/main.rb:55]: Yes you do not need this puts statement i guess

**Pull request merged**